sdk/ml/azure-ai-ml/azure/ai/ml/_schema/pipeline/pipeline_component.py

```diff
@@ -107,6 +108,10 @@ def _post_load_pipeline_jobs(context, data: dict) -> dict:
                     loaded_data=job_instance,
                 )
                 jobs[key] = job_instance
+            elif job_instance.get("type") == ControlFlowType.IF_ELSE:
+                # Convert to if-else node.
+                job_instance = ConditionNode._create_instance_from_schema_dict(loaded_data=job_instance)
+                jobs[key] = job_instance
```

Comment:
seems like many duplicate line:

jobs[key] = job_instance

Reply:
Good catch! Will fix this in next PR